source/options.rst

```diff
@@ -0,0 +1,90 @@
+.. _operator-configmaps:
+
+Changing PostgreSQL Options
+===========================
+
+You may require a configuration change for your application. PostgreSQL
+allows the option to configure the database with a configuration file.
+You can pass the `PostgreSQL configuration options <https://www.postgresql.org/docs/9.3/config-setting.html>`__
+in one of the following ways:
+
+* edit the ``deploy/cr.yaml`` file,
+* use a ConfigMap.
+
+.. _operator-configmaps-cr:
+
+Edit the ``deploy/cr.yaml`` file
+---------------------------------
+
+You can add options by editing the configuration section of the
+``deploy/cr.yaml``. Here is an example:
+
+.. code:: yaml
+
+   spec:
+     ...
+     pgPrimary:
+       ...
+         customconfig: |
```

Comment:
in other operators we have `configuration` not `customconfig`. Is it something we inherited @hors  ?